edb/server/pgconnparams.py

```diff
@@ -48,7 +48,12 @@ def get_pg_home_directory() -> pathlib.Path:
 else:
 
     def get_pg_home_directory() -> pathlib.Path:
-        return pathlib.Path.home() / '.postgresql'
+        try:
+            return pathlib.Path.home() / '.postgresql'
+        except RuntimeError:
+            # In some container environments the home dir is not
+            # available so we return a path to a nonexistent file
+            return pathlib.Path('/.postgresql')
```

Comment:
This (maybe load keys from `/.postgresql` if it unexpectedly exists) is different than what libpq does upstream (don't load anything if homedir isn't available):

https://github.com/postgres/postgres/blob/2247281c470502c799fc2153a3583c025e894a9c/src/interfaces/libpq/fe-secure-openssl.c#L1149-L1159

Ideally, we should make `home_dir` an optional value here:

https://github.com/geldata/gel/blob/5c9955b4e6197ca677ebaa4d1b3867fad3b1d288/rust/pgrust/src/python/mod.rs#L280

Reply:
I changed the code to return an empty path, however I don't know any rust so I would need help to change that part ...